Essentials/src/main/java/com/earth2me/essentials/commands/Commandbroadcastworld.java

```diff
@@ -39,7 +39,7 @@ public void run(final Server server, final User user, final String commandLabel,
     @Override
     public void run(final Server server, final CommandSource sender, final String commandLabel, final String[] args) throws Exception {
         if (args.length < 2) {
-            throw new NotEnoughArgumentsException("world");
+            throw new NotEnoughArgumentsException();
```

Comment:
Why remove this?

Reply:
Checked the blame, looks like it was an oversight in the originally approval, there is no reason to send the string "world" here, literally only need to send the command usages.